packages/kbn-eslint-config/.eslintrc.js

```diff
@@ -115,6 +115,12 @@ module.exports = {
           disallowedMessage: `import from @kbn/i18n-react instead`,
           exclude: [/packages[\/\\]kbn-i18n-react[\/\\]/],
         },
+        {
+          from: 'zod',
+          to: '@kbn/zod',
+          disallowedMessage: `import from @kbn/zod instead`,
+          exclude: [/packages[\/\\]kbn-zod[\/\\]/],
```

Comment:
Here is what the error will look like:

![image](https://github.com/user-attachments/assets/fb56bc5a-d733-4c28-b378-9d657cc54166)